### DISCUSSION TITLE
any interest to use the "new" issue template form format on the handbook ?

### DISCUSSION THREAD

**Discussion by Remi-Gau** [datalad-handbook/book#766] [Ideas]
I have started setting those ups on some repos here and there and the results looks nicer and less scary to non github user than the "traditional" issues.

Here is a sample: https://github.com/brainhackorg/global2021/issues/new?assignees=&labels=website&template=website-issue-template.yml

 I could set those up for the handbook if there is some interest.

**Comment by adswa** [comment]
This is super cool, how do keep finding these awesome features with such frequency? They would be a great addition, I would be very happy about such a PR :-) And I immediately know of more places where this would be useful. @yarikoptic , @mih : I think datalad itself would benefit from it. What do you think?

**Reply by Remi-Gau** [reply]
> how do keep finding these awesome features with such frequency? 

Simple: 

1. I don't have a life
2. I am on too many:
- mailing lists, 
- slack workspaces, 
3. I follow too many:
  -  youtube channels
  - podcasts

> They would be a great addition, I would be very happy about such a PR :-) And I immediately know of more places where this would be useful. @yarikoptic , @mih : I think datalad itself would benefit from it. What do you think?

If you think that you can use the same template across several repos of an orga, then there is the option of putting those in a `.github` repo.

https://github.com/cpp-lln-lab/.github

Am pretty sure you can still override those orga wide templates by having a template specific templates with the same name.

How about open a PR with a suggestion and let you all go wild from there ?

**Comment by adswa** [comment]
(LOL to the start of your comment, I have too little time to appropriately respond atm)

> How about open a PR with a suggestion and let you all go wild from there ?

Sounds superb. If you're up to contributing to DataLad too (we are not the same orga, the handbook is a strong and independent git repo and has its own orga), I'm sure github.com/datalad/datalad would be happy to see the PR as well :blush:

**Reply by mih** [reply]
I would be very curious, indeed!
